src/odemis/odemisd/main.py

```diff
@@ -518,10 +518,12 @@ def set_base_group(self):
         try:
             os.setgid(gid_base)
         except OSError:
+            # This can happen especially when running the backend as a standard user.
             logging.warning("Not enough permissions to run in group " + model.BASE_GROUP + ", trying anyway...")
 
-        # everything created after must be rw by group
-        os.umask(~(stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IWGRP))
+        # Everything created after should be (also) accessible by the group.
+        # Need the user execute bit, to allow directory creation (with files inside)
+        os.umask(~(stat.S_IRWXU | stat.S_IRGRP | stat.S_IWGRP))
```

Comment:
is it possible/does it make sense to add a test case for this, or is this already tested?

Reply:
Well, without it, a lot of test cases failed to run on Ubuntu 20.04, including most of the ones in odemisd/test/. So it was indirectly tested. There is nothing explicitly checking that the files created in /var/run/odemisd/ are forbidden to the non odemis group, but I don't think it's super important to check per-se.